.travis.yml

```diff
@@ -1,9 +1,8 @@
 language: node_js
 node_js:
-  - 0.6
-  - 0.8
```

Comment:
The origin file is 5.86kB by the way, so this even got worse by `-p`. I start to wonder if this is even worth the hassle.